content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+content_type: topic
+prerequisites:
+  hard:
+  - topics/file-and-directory-naming/general
+ready: true
+tags:
+- File and directory naming
+- python
+title: File and directory naming in Python
+---
+
+## How should I name my Python modules and packages?
+
+You should name your files and directories the same way you name Python variables.They should be small letters, and words should be separated by underscores, as that improves readability.
+
+It is important to note that the use of dashes in directory-naming takes preference as opposed to the use of underscores and be sure to keep in mind that all Python files have a .py extension. See below the recommended Python file and directory naming style. 
+
+#### NO:
+```
+TASK/
+    1task.py
+    task 1.py
+    task1.1.py
+        
+passwordChecker/
+    add-logger.py
+
+snakecase/
+    productionnotes.py
+    keepItShortAndmeaningful.py
+```
+
+#### Yes:
+
+```
+task/
+    task_1.py
+    task_2.py
+    task_1_1.py
+
+password-checker/
+    add_logger.py
+
+snake_case/
```

Comment:
```suggestion
snake-case/
```

Reply:
this is a directory

Reply:
According to my understanding, for directory naming , underscores are accepted , but dashes take preference.

Reply:
<img width="823" alt="Screenshot 2022-03-28 at 11 22 04" src="https://user-images.githubusercontent.com/50909846/160367897-50ed909f-6c62-4067-9509-8ebe3d2b7535.png">

Reply:
this is how it renders on my pc. 
So just need to fix so that the formatting is consistent